topology/University of Campinas/CCJDR/feynman-PBS.yaml

```
@@ -0,0 +1,119 @@
+
+# If you have an up-to-date local git clone, fill GroupID with the output from `bin/next_resource_group_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+GroupID:
+
+# Resources contains one or more resources in this
+# ResourceGroup. A resource provides one or more services
+Resources:
+  # Resource Name should be a short descriptor of the resource.
+  # e.g. the Center for High Throughput Computing's GlideinWMS Frontend is "CHTC-glidein2"
+  # Resource Names need to be unique across all resources in the OSG.
+  feynman:
+    # Active is true if the resource is accepting requests, and false otherwise.
+    # When first registering a resource, set this to false. Set it to true when it's ready for production.
+    Active: false
+    # Description is a long description of the resource; may be multiple lines
+    Description: This is a HPC CLuster for CCJDR to used by Dune
+    # If you have an up-to-date local git clone, fill ID with the output from `bin/next_resource_id`
+    # Otherwise, leave it blank and we will fill in the appropriate value for you.
+    ID:
+    # ContactLists contain information about people to contact regarding this resource.
+    # The "ID" is a hash of their email address available at https://topology.opensciencegrid.org/miscuser/xml
+    # If you cannot find the contact above XML, please register the contact:
+    # https://opensciencegrid.org/docs/common/registration/#registering-contacts
+    ContactLists:
+      # Administrative Contact is one to three people to contact regarding administrative issues
+      Administrative Contact:
+        Primary:
+          Name: Fabio Andrijauskas
+          ID: ef5d0be3ec1df913935853ed82396b1eb37a7db0
+        # Secondary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+        # Tertiary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+
+      # Security Contact is one to three people to contact regarding security issues
+      Security Contact:
+        Primary:
+          Name: Ernesto Kemp
+          ID:  7c8a781a5c0042d6f90ffc3d64e1b9547a5f2f18
+        # Secondary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+        # Tertiary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+
+    # FQDN is the fully qualified domain name of the host running this resource
+    FQDN: condor.ifi.unicamp.br
+    ### FQDNAliases (optional) are any other DNS aliases by which this host can be accessed
+    # FQDNAliases:
+    #   - <HOSTNAME1>
+    #   - <HOSTNAME2>
+
+    ### DN (optional except for XCache resources) is the DN of the host cert of the resource
+    # in OpenSSL 1.0 format (i.e. /DC=org/DC=incommon/C=US/...)
+    # DN: <DN>
+
+    # Services is one or more services provided by this resource;
+    # valid services are listed in topology/services.yaml with the format "<SERVICE NAME>: <ID>"
+    Services:
+      <SERVICE NAME>:
```

```suggestion
      CE:
```

From the FQDN, I gather that this is an HTCondor-CE?

---

condor.ifi.unicamp.br

---

Thanks for the response @biozit. What grid services do you have installed on `condor.ifi.unicamp.br`? Is it an HTCondor-CE?

---

The HTCondor-CE is installed on condor.ifi.unicamp.br, however, the jobs will be routed to another host,